bin/delete_duplicate_draft_manuals_and_draft_manual_sections

```diff
@@ -2,7 +2,7 @@
 
 require File.expand_path("../../config/environment", __FILE__)
 
-["c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e c23dc478-cd8f-437b-aba2-2cee9aed60e2", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e ecce885d-7225-4fd4-b23f-da0d0eee9849", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 0e52e6b8-8d81-4f9f-a139-4af7acc5ec71", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 1be90806-e94d-4ece-a40f-c040b21ea7d0", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 97f16fee-0eff-452e-aecc-d0c55126f328", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 9c12c55e-00aa-4417-8c6b-a1314cf1fff7", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e f3030447-158a-4821-906b-7e25027ac742", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 911e7471-0760-4bf0-a113-e833e7dca042", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e f97c7969-3ccc-42b1-bc03-c270e08c15a2", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 7f0447b6-d3e4-4716-8527-dbbb3104a0a3", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 6cada3f6-8e45-45cb-8a39-e9e70fbfdafa", "39e0c495-64b3-48c3-8905-30ee4c46867c 70ed29bc-f267-4828-b94c-2dba63a498ef", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e ae86a4be-3bc5-4d53-b4ed-df4f73fa8a91", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e e59f0d99-5566-4b9e-bcc5-dd26c8686664"].each do |i|
+["c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e c23dc478-cd8f-437b-aba2-2cee9aed60e2", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e ecce885d-7225-4fd4-b23f-da0d0eee9849", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 0e52e6b8-8d81-4f9f-a139-4af7acc5ec71", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 1be90806-e94d-4ece-a40f-c040b21ea7d0", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 97f16fee-0eff-452e-aecc-d0c55126f328", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 9c12c55e-00aa-4417-8c6b-a1314cf1fff7", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e f3030447-158a-4821-906b-7e25027ac742", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 911e7471-0760-4bf0-a113-e833e7dca042", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e f97c7969-3ccc-42b1-bc03-c270e08c15a2", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 7f0447b6-d3e4-4716-8527-dbbb3104a0a3", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e 6cada3f6-8e45-45cb-8a39-e9e70fbfdafa", "39e0c495-64b3-48c3-8905-30ee4c46867c 70ed29bc-f267-4828-b94c-2dba63a498ef", "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e e59f0d99-5566-4b9e-bcc5-dd26c8686664"].each do |i|
```

Comment:
The diff is hard to read, but this is the only pair of content-ids that is deleted "c3ed3b6b-235b-4cae-869c-a5d65fcb3f7e ae86a4be-3bc5-4d53-b4ed-df4f73fa8a91"